eras/shelley-ma/test-suite/cardano-ledger-shelley-ma-test.cabal

```
@@ -54,7 +54,7 @@ library
         containers,
         hashable,
         cardano-ledger-shelley-test >=1.1,
-        cardano-ledger-shelley:{cardano-ledger-shelley, testlib} ^>=1.3,
+        cardano-ledger-shelley:{cardano-ledger-shelley, testlib} ^>=1.4,
```

Also needs a patch bump
```suggestion
        cardano-ledger-shelley:{cardano-ledger-shelley, testlib} >=1.3 && <1.5,
```